DB/projects.json

```diff
@@ -3634,7 +3634,23 @@
           "API",
           "Vercel"
         ]
+      },
+      {
+        "link": "https://github.com/Harshu467/NFT",
+        "title": "NFT Market",
+        "description": "Master React Native by building a modern NFT Marketplace iOS and Android #ReactNative Application ",
+        "tech": [
+          "html",
+          "css",
+          "javascript",
+          "reactjs JS",
+          "useContext",
+          "API",
+          "Vercel",
```

Comment:
just you need to mention tech stacks (e.g. reactjs, nodejs, not vercel, usecontext  api )
```suggestion
    
```